backend/PostProcessing/IPostProcessing.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+#IPostProcessing.py
+#----------------------------------
+# Created By: Matthew Kastl
+#----------------------------------
+"""Post processing is responsible for preforming operation on data within the collection of data. 
+An example of this could be preforming an arithmetic operation on a column of data or cleaning missing values.)
+This module provides an interface to decouple this process from the main body of code. The factory method will 
+import and execute the correct code for a given process (based on a keyword) returning the result.
+New data will be inserted into the DataFrame by reference
```

Comment:
what if there was a period at the end of this sentence 👀

Reply:
What if indeed!